src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,40 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final String PACKAGE_NAME = "mate.academy";
+    private static final String BOOK_NAME_1 = "Clean code";
+    private static final String BOOK_NAME_2 = "Spring in Action";
+    private static final String BOOK_NAME_3 = "Test-Driven Java Development";
+    private static final BigDecimal BOOK_PRICE_1 = BigDecimal.valueOf(25.33);
+    private static final BigDecimal BOOK_PRICE_2 = BigDecimal.valueOf(39.51);
+    private static final BigDecimal BOOK_PRICE_3 = BigDecimal.valueOf(49.99);
+    private static final BigDecimal BOOK_PRICE_NEW = BigDecimal.valueOf(27.20);
+    private static final Injector injector = Injector.getInstance(PACKAGE_NAME);
```

Comment:
You declare a constant that is used to declare a constant and you use it only one time, I think we can avoid constant declaration here.
```suggestion
    private static final Injector injector = Injector.getInstance("mate.academy");
```